app/models/entity.rb

```diff
@@ -0,0 +1,4 @@
+class Entity < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Please ensure that you add validation to your Entity model to make sure that anyone with bad intentions cannot compromise your app.

Reply:
Oh, I will improbe it by adding validations, thanks